tests/test_paperqa.py

```diff
@@ -2616,3 +2622,49 @@ def test_pqa_context_id_parsing(raw_text: str, cleaned_text: str) -> None:
     )
     session.populate_formatted_answers_and_bib_from_raw_answer()
     assert session.answer == cleaned_text
+
+
+@pytest.mark.skipif(os.environ.get("ANTHROPIC_API_KEY") is None, reason="No API key")
```

Comment:
```suggestion
```

Let's just always run this test

Reply:
You sure? I was copying what we did for `test_anthropic` above

Reply:
From my experience having disabled aspects of CI can lead to hidden failures

I am okay if you'd like to keep it though